contracts/BlankArt.sol

```diff
@@ -304,7 +304,7 @@ contract BlankArt is ERC721, EIP712, ERC721URIStorage, Ownable, IERC2981 {
     }
 
     /// @notice Transfers all pending withdrawal balance to the caller. Reverts if the caller is not an authorized minter.
-    function withdraw() public onlyOwner {
+    function withdraw() public {
```

Comment:
I may be missing something, but wouldn't this allow anyone to withdraw the funds? Wouldn't it be better to leave this as the owner of the address, but change the receiver to always be the foundation address?

Or are we trying to allow people to withdraw their own funds?

Very sorry if my lack of solidity expertise if leading me astray here...

Reply:
@asebesta explained this to me. Makes sense now.